docs/checkit.js

```diff
@@ -460,17 +458,11 @@ CheckIt.prototype.generateCheckmarks = function(calObj, $div) {
      })
 };
 
-CheckIt.prototype.removeEmptyWeeks = function(calObj, $div) {
+CheckIt.prototype.removeEmptyWeeks = function(calObj, $calendarDiv) {
     // Remove empty weeks from the calendar
         
-    $div.find('.month').find('.week').each( function(index) {
-        var counter = 0;
-        $(this).find('td').each(function(td) {
-            if ($(this).children().hasClass("nill")) 
-            counter +=1;
-        })
-        if (counter === 7) 
-        {
+    $calendarDiv.find('.month').find('.week').each( function(index) {
```

Comment:
These are CSS selectors, right? I don't think you should have to do multiple calls in a row. `find('.month > .week')`